<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(60, 43, 50);
font: 24px 'Arial Rounded MT Bold';
color: #7988a8;}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>371</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="GameModeSelect">
        <property name="styleSheet">
         <string notr="true">color: rgb(145, 111, 125);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="HumanTimeEdit">
        <property name="styleSheet">
         <string notr="true">color: rgb(145, 111, 125);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#916f7d;&quot;&gt;Bot&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#916f7d;&quot;&gt;时限&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#916f7d;&quot;&gt;(s)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#916f7d;&quot;&gt;Human&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#916f7d;&quot;&gt;时限&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#916f7d;&quot;&gt;(s)   &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="BotTimeEdit">
        <property name="styleSheet">
         <string notr="true">color: rgb(145, 111, 125);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#916f7d;&quot;&gt;Game Type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#916f7d;&quot;&gt;先手方&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="FirstPlayerSelect">
        <property name="styleSheet">
         <string notr="true">color: rgb(145, 111, 252);
color: rgb(145, 111, 125);
selection-color: rgb(145, 111, 125);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="OKPushButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(145, 111, 125);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
